store/storetest/post_store.go

```diff
@@ -1976,7 +1976,7 @@ func testPostStoreGetPostsBatchForIndexing(t *testing.T, ss store.Store) {
 	o3.Message = "zz" + model.NewId() + "QQQQQQQQQQ"
 	o3 = (<-ss.Post().Save(o3)).Data.(*model.Post)
 
-	if r := store.Must(ss.Post().GetPostsBatchForIndexing(o1.CreateAt, model.GetMillis()+100000, 100)).([]*model.PostForIndexing); len(r) != 3 {
+	if r, _ := ss.Post().GetPostsBatchForIndexing(o1.CreateAt, model.GetMillis()+100000, 100); len(r) != 3 {
```

Comment:
Would you please check the return here?

Reply:
Thank you for your feedback @hanzei, I have updated my PR to check the error return value here. I think that's what you wanted, but if not please let me know!